energi_service/energi_service_plot/plot_gridarea_data.py

```diff
@@ -0,0 +1,101 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+
+
+def read_csv(path: str) -> pd.DataFrame:
```

Comment:
Should rebase with main and use the function read_csv from the util.util module instead.